.github/workflows/gateway.yml

```diff
@@ -110,7 +110,48 @@ jobs:
         with:
           flags: integrationtests #optional
 
-  build:
+  system-tests:
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        python: [3.9]
```

Comment:
no 3.8 version?

Reply:
no. this is due to the fact that piwheels does only provide pre-built libraries for python 3.7 and 3.9. so I went for the more recent.